### PR TITLE
Fix duplicate IDs in mobile menu

### PR DESCRIPTION
**Pull request by dcmouyard**

### CODE REVIEW

File: source/03-components/mobile-menu/modules/_MobileMenu.es6.js [hunk 1]
```diff
@@ -76,6 +76,14 @@ class MobileMenu extends OverlayMenu {
     if (blockClone.id) {
       blockClone.id = `${blockClone.id}-mobile`;
     }
+    const childrenWithId = [...blockClone.querySelectorAll('[id]')];
+    if (blockClone.matches('[id]')) {
+      childrenWithId.push(blockClone);
+    }
```
Comment (kmonahan) [inline source/03-components/mobile-menu/modules/_MobileMenu.es6.js]: I like how you're handling the children, but this seems like it duplicates what's right above it.
```
    if (blockClone.id) {
      blockClone.id = `${blockClone.id}-mobile`;
    }
```
Can lines 80-82 be removed?

Comment (dcmouyard) [reply source/03-components/mobile-menu/modules/_MobileMenu.es6.js]: Good catch! Removed redundant code.

**Comment by dcmouyard** [comment]
I added some code to update the `for` attribute of elements to match the updated IDs.

**Review by kmonahan — APPROVED** [review]

**Comment by kmonahan** [comment]
@coreylafferty Ready to be merged for the next release. Does not apply to GUS or Gesso WP.

**Pull request merged**